bedrock/foundation/templates/foundation/index.html

```diff
@@ -32,6 +32,12 @@ <h1 class="page-title">{{ _('The Mozilla Foundation') }}</h1>
   {% endtrans %}
   </p>
 
+  <p>
+    {% trans url_open_strategy_document='https://mzl.la/foundation-strategy' %}
```

Comment:
You can isolate the text inside the link so all that surrounding HTML isn't part of the string. Then you can also just put the URL in directly and forego the trans block:

`<p><strong><a href="https://mzl.la/foundation-strategy" class="go">{{ _('Read the Mozilla Foundation’s 2016-2018 Strategy') }}</a></strong></p>`